onward/app/controllers/MostPopularController.scala

```diff
@@ -37,6 +37,7 @@ object MostPopularController extends Controller with Logging with ExecutionConte
         case popular => Cached(900) {
           JsonComponent(
             "html" -> views.html.fragments.collections.popular(popular),
+            "ABhtml" -> views.html.fragments.collections.popularABTest(popular),
```

Comment:
Adding this extra blob of HTML to the JSON response will add around 2kb gzipped.